backend/app/services/webhook_notification.py

```
@@ -0,0 +1,181 @@
+                    self.endpoint_url,
+                    json=payload,
+                    headers=headers,
+                    timeout=self.timeout
+                )
+            else:
+                logger.error(f"Unsupported HTTP method: {self.http_method}")
+                return False
+
+            response.raise_for_status()
+            logger.info(f"Webhook notification sent successfully for {notification.event} id={notification.id}")
+            return True
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)